cloud/storage/core/libs/common/compressed_bitmap.cpp

```diff
@@ -83,21 +84,21 @@ class TCompressedBitmap::TImpl
             auto rangeInfo = NPrivate::RangeInfo<ui16, 64>(b, e);
 
             if (rangeInfo.First == rangeInfo.Last) {
-                count -= PopCount(Data[rangeInfo.First]);
+                count -= std::popcount(static_cast<ui64>(Data[rangeInfo.First]));
```

Comment:
let's remove the `PopCount` method from `TCompressedBitmap::TImpl` if it's not used anymore

Reply:
fixed